src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallService {
+
+    public CinemaHall add(CinemaHall cinemaHall);
+
+    public CinemaHall get(Long id);
+
+    public List<CinemaHall> getAll();
```

Comment:
The `public` modifier is redundant in interface method declarations. All methods in an interface are implicitly public, so you can remove the `public` keyword from the method declarations.